finagle-core/src/main/java/com/twitter/finagle/ssl/KeyCredentialsConfig.java

```diff
@@ -33,4 +34,11 @@ public static KeyCredentials certKeyAndChain(
     return new KeyCredentials.CertKeyAndChain(certificateFile, keyFile, caCertificateFile);
   }
 
+  /**
+   * See {@link KeyCredentials.FromKeyManager}
+   * @see KeyCredentials.FromKeyManager // Refactor to use @see instead?
+   */
+  public static KeyCredentials fromKeyManagerFactory(KeyManagerFactory keyManagerFactory) {
```

Comment:
i think the naming pattern in this file would make this method be called "keyManagerFactory`.

Reply:
Done